jenkins-master/jobs/trigger-ondemand/workspace/trigger.py

```diff
@@ -63,7 +63,6 @@ def get_test_packages_url(properties):
         'locale': 'en-US',
         'extension': 'test_packages.json',
         'build_type': 'tinderbox',
-        'retry_attempts': 0,
```

Comment:
We have to retry finding the new file name before we use the fallback. Just in case there is a delay in uploading.